ledger/sandbox-on-x/reference.conf

```diff
@@ -6,10 +6,21 @@
 # Commented out sections or fields are optional.
 # Some values choosen here as defaults are no intended to be production ready.
 
+# TODO: one of the bigger questions is: parameter vs functional control. in Canton, we are keeping the expert knobs in canton.parameters or canton.participant.p.parameters
+#       and thus separate it from the config options that a user would normally change. at some point, we actually wanted to say that setting parameters is "unsupported".
+#       generally, default parameters should just be chosen such that they never have to be changed or tuned by a normal user.
+#       of course, this isn't 100% clean and consistent in canton so far. but at least, that was the idea.
+#       in the api server config, we do this very much different. the config structure corresponds to the component and the configurations are very close to that.
+
 ledger {
   # The ID of the ledger. This must be the same each time the ledger is started.
   ledger-id = default-ledger-id
 
+  # TODO consolidate with Canton: 
```

Comment:
IMO metrics setup should not be configurable for ledger server when it is embedded into canton. Instead the metrics registry should be passed to the ledger server constructor. In this way we kill 2 birds with one stone: we use canton metrics config with its concurrent reporters and ledger server metrics appear together with canton metrics in the same reporting channels.

Reply:
We might want though to lift the canton metrics implementation into sdk (or merge if there are things worth preserving from sdk)